backend/src/modules/doubles-tournament/core/application/event/ReleaseTableInQueue.ts

```diff
@@ -0,0 +1,25 @@
+import { TournamentTableWasReleased } from '../../../../tournament-tables/core/domain/event/TournamentTableWasReleased';
+import { EventHandler } from '../../../../../shared/core/application/event/EventHandler';
+import { TablesQueueRepository } from '../TablesQueueRepository';
+import { pushTableToQueue } from '../../domain/QueuedTable';
+import { TournamentId } from '../../domain/TournamentId';
+
+export class ReleaseTableInQueue implements EventHandler<TournamentTableWasReleased> {
+  private readonly tablesQueueRepository: TablesQueueRepository;
+
+  constructor(tablesQueueRepository: TablesQueueRepository) {
+    this.tablesQueueRepository = tablesQueueRepository;
+  }
+
+  async handle(event: TournamentTableWasReleased): Promise<void> {
```

Comment:
Chociaż teraz już nie jestem pewien... czy bazujesz na kolejności event handlerów? Jak to jest teraz, że w 2 różnych jest dodawanie stołu do kolejki i wywoływanie meczu na ten sam event?